app/views/spree/orders/edit.html.erb

```diff
@@ -1,6 +1,6 @@
 <% @body_id = 'cart' %>
 
-<article class="cart-page">
+<article class="cart-page" data-hook="cart_container">
```

Comment:
why do we need this?

Reply:
We said that we should open an issue on solidus_stripe to add instructions on how to decorate views without using deface and data-hooks (that we want totally discard from the new code)

Reply:
As discussed IRL, let's get rid of this. The stripe documentation should not mention deface.